conanfile.py

```diff
@@ -16,8 +16,9 @@ class ProxyFmuConan(ConanFile):
     generators = "cmake"
     requires = (
         "cli11/2.3.1",
+        "boost/1.81.0",
```

Comment:
Need to specify boost version here otherwise libcosim won't build unless the version matches exactly what thrift uses (which changes over time).